src/main/java/org/openpnp/machine/reference/ReferenceMachine.java

```diff
@@ -298,4 +304,20 @@ public PnpJobProcessor getPnpJobProcessor() {
     public PasteDispenseJobProcessor getPasteDispenseJobProcessor() {
         return pasteDispenseJobProcessor;
     }
+
+    public boolean getHomeAfterEnabled() {
+        return homeAfterEnabled;
+    }
+
+    public boolean getUsePickRotationInsteadOfRotationInTapeForStripFeeders() {
+        return usePickRotationInsteadOfRotationInTapeForStripFeeders;
+    }
+
+    public void setHomeAfterEnabled(boolean newValue) {
+        this.homeAfterEnabled = newValue;
+    }
+
+    public void setUsePickRotationInsteadOfRotationInTapeForStripFeeders(boolean newValue) {
```

Comment:
i place there, to change this setting for ALL strip feeders. If we put it in feeder wizard, it will apply only for 1 feeder, i think.

Reply:
Yes, it will only apply to one. We could add it as a machine property, and set it as a default from within the strip feeder, but it needs to live there one way or another. I won't put feeder implementation specific settings in the machine UI. It is a recipe for clutter.